react/components/atoms/lightbox/lightbox.js

```diff
@@ -0,0 +1,91 @@
+import React, { PureComponent } from "react";
+import { ViewPropTypes, StyleSheet, View, TouchableOpacity, Image, Modal } from "react-native";
+
+import PropTypes from "prop-types";
+
+export class Lightbox extends PureComponent {
+    static get propTypes() {
+        return {
+            src: PropTypes.string.isRequired,
+            width: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+            height: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+            borderRadius: PropTypes.number,
+            resizeMode: PropTypes.string,
+            style: ViewPropTypes.style
+        };
+    }
+
+    static get defaultProps() {
+        return {
+            src: undefined,
+            width: "100%",
+            height: "100%",
+            borderRadius: undefined,
+            resizeMode: undefined
+        };
+    }
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            visible: false
+        };
+    }
+
+    onBackButtonPress = () => {
+        this.setState({
+            visible: false
+        });
+    };
+
+    onLightboxPress = () => {
+        this.setState({
+            visible: true
+        });
+    };
+
+    _imageStyle = () => {
+        return [
+            {
+                width: this.props.width,
+                height: this.props.height,
+                borderRadius: this.props.borderRadius,
+                resizeMode: this.props.resizeMode
+            },
+            this.props.style
+        ];
+    };
+
+    render() {
+        return (
+            <View>
+                <TouchableOpacity onPress={() => this.onLightboxPress()} activeOpacity={0.7}>
+                    <Image style={this._imageStyle()} source={{ uri: this.props.src }} />

```

Comment:
We need to potentially allow other kind of sources, so make:
- Rename `src` to `uri`
- Add a `source` prop
- Use `src` if defined, else use `source`